Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,42 @@
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.10.6.tar.gz"
+  sha256 "01ad50677e2bde3bcd51128a9e043f76371e9642a5bea235637ce8e450eec5de"
+  head "https://github.com/aws/aws-sdk-cpp.git"
+
+  option "with-custom-memory-management", "Build with custom memory management"
```

Comment:
"If static linking is enabled, custom memory management defaults to off. If dynamic linking is enabled, custom memory management defaults to on and avoids cross-DLL allocation and deallocation."

Sounds like this should just be omitted as an option if it's turned on/off like this to avoid confusion.